pkg/ui/src/index.tsx

```diff
@@ -110,6 +111,7 @@ ReactDOM.render(
           <Route path={`range/:${rangeIDAttr}/cmdqueue`} component={ CommandQueue } />
         </Route>
         { visualizationRoutes() }
+        <Route path="*" components={ NotFound } />
```

Comment:
why "components" not "component? are they equivalent?

Reply:
Whoops, typo. Don't know why that even worked :P

Reply:
also aren't you on vacation? haha

Reply:
#naptime

Reply:
lol. TFTR!